clean/full_clean_C3560CX_ios/c3560cx_clean.yaml

```diff
@@ -0,0 +1,125 @@
+cleaners:
+  DeviceClean:
+    module: genie.libs.clean
+    devices: [3560cx]
+
+devices:
+  3560cx:
+    device_recovery:
+      break_count: 10
+      console_activity_pattern: "\\.\\.\\.\\."
+      timeout: 1200
+      recovery_password: lab
+      golden_image:
+        - flash:/c3560cx-universalk9-mz.152-7.E1/c3560cx-universalk9-mz.152-7.E1.bin
+      powercycler: True
+
+    connect:
+
+    copy_to_device:
+      origin:
+        files:
+          - mypath/test.cfg
+        hostname: 10.15.19.10
+      destination:
+        directory: 'flash:/'
+      protocol: ftp
+      check_file_stability: True
+      min_free_space_percent: 50
+      # if_exist_still_copy: True
```

Comment:
I would recommend taking this out or changing it to use the actual flag, `overwrite: True`